__tests__/pages/500.test.js

```diff
@@ -1,3 +1,4 @@
+import '../../__mocks__/next-auth/nextAuthAPI.mock'
```

Comment:
Why is it required in 500 page?

Reply:
Because 500 page uses [`Layout`](https://github.com/garageScript/c0d3-app/blob/master/components/Layout.tsx) component which renders [`Appnav`](https://github.com/garageScript/c0d3-app/blob/master/components/AppNav.tsx) which uses Next auth `useSession` API.